system/modules/core/forms/FormTextArea.php

```diff
@@ -144,7 +144,7 @@ public function __get($strKey)
 				break;
 
 			case 'value':
-				return specialchars(str_replace('\n', "\n", $this->varValue));
+				return str_replace('\n', "\n", $this->varValue);
```

Comment:
We are just restoring the previous behavior, because the last change was incorrect.